rules/windows/execution_via_hidden_shell_conhost.toml

```
@@ -14,7 +14,65 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Conhost Spawned By Suspicious Parent Process"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating Conhost Spawned By Suspicious Parent Process
+
+The Windows Console Host, or `conhost.exe`, is both the server application for all of the Windows Console APIs as well as
+the classic Windows user interface for working with command-line applications.
+
+Attackers often rely on custom shell implementations to avoid using built-in command interpreters like `cmd.exe` and 
+`PowerShell.exe` to bypass application whitelisting and security features and commonly inject these implementations into
+legitimate system processes.
```

I agree with @nastasha-solomon's comment here; the sentence is long and would benefit by breaking up or restructuring somehow. This is a slight modification of her suggestion:
```suggestion
Attackers often rely on custom shell implementations to avoid using built-in command interpreters like `cmd.exe` and 
`PowerShell.exe` and bypass application allowlisting and security features. Attackers commonly inject these implementations into
legitimate system processes.
```